src/tokens/Assets.tsx

```diff
@@ -483,6 +484,14 @@ function AssetsScreen({ navigation, route }: Props) {
 
 AssetsScreen.navigationOptions = {
   ...headerWithBackButton,
+  headerRight: HeaderRight,
+}
+
+function HeaderRight() {
+  if (getFeatureGate(StatsigFeatureGates.SHOW_IMPORT_TOKENS_FLOW)) {
+    return <ImportTokenButton />
```

Comment:
i don't think you need a new component at all, you can directly reuse `TopBarTextButton`. [here](https://github.com/valora-inc/wallet/blob/9fa5c4d87373843306815be395f4f54c36bd8fa3/src/webview/WebViewScreen.tsx#L95-L100) is an example